packages/core/test/actions.test.ts

```diff
@@ -1054,106 +1054,127 @@ describe('action meta', () => {
 });
 
 describe('purely defined actions', () => {
-  interface Ctx {
-    items: Array<{ id: number }>;
-  }
-  type Events =
-    | { type: 'SINGLE'; id: number }
-    | { type: 'NONE'; id: number }
-    | { type: 'EACH' };
-
-  const dynamicMachine = Machine<Ctx, Events>({
-    id: 'dynamic',
-    initial: 'idle',
-    context: {
-      items: [{ id: 1 }, { id: 2 }, { id: 3 }]
-    },
-    states: {
-      idle: {
-        on: {
-          SINGLE: {
-            actions: pure<any, any>((ctx, e) => {
-              if (ctx.items.length > 0) {
-                return {
-                  type: 'SINGLE_EVENT',
-                  length: ctx.items.length,
-                  id: e.id
-                };
-              }
-            })
-          },
-          NONE: {
-            actions: pure<any, any>((ctx, e) => {
-              if (ctx.items.length > 5) {
-                return {
-                  type: 'SINGLE_EVENT',
-                  length: ctx.items.length,
-                  id: e.id
-                };
-              }
-            })
-          },
-          EACH: {
-            actions: pure<any, any>((ctx) =>
-              ctx.items.map((item: any, index: number) => ({
-                type: 'EVENT',
-                item,
-                index
-              }))
-            )
+  describe('Declaring actions as { type: string }', () => {
```

Comment:
Moved this into its own describe block

Yes, I know

https://twitter.com/AndaristRake/status/1512370757669695491